### DISCUSSION TITLE
ko-kr/manual/device/windows

### DISCUSSION THREAD

**Discussion by giscus** [MaaAssistantArknights/maa-website#153] [Comments]
# ko-kr/manual/device/windows

Windows 지원 아래의 목록은 무작위순이며, 성능순이 아닙니다. ✅ 완벽한 지원 ⚠️ 부분 지원 🚫 미지원

https://docs.maa.plus/ko-kr/manual/device/windows.html

**Comment by Jong-BinPark** [comment]
부분지원의 뮤뮤와 완전지원의 뮤뮤는 서로 다릅니다
부분지원은 뮤뮤6 이고 완전지원은 그냥 뮤뮤
다른 언어는 다 다른데 한국어만 같아서 뭔가 했네요
